### PR TITLE
modulo arithmetic unit tests

### PR DESCRIPTION
**Pull request by omershlo**
#6

### CODE REVIEW

File: src/arithmetic/big_gmp.rs [hunk 1]
```diff
@@ -128,4 +129,40 @@ mod tests {
...
         }
     }
+    //test mod_sub: a-b mod n where a-b >0
+    #[test]
+    fn test_mod_sub_modulo() {
+        let a = Mpz::from(10);
+        let b = Mpz::from(5);
+        let modulo = Mpz::from(3);
+        let res = Mpz::from(2);
+        assert_eq!(res,Mpz::mod_sub(&a,&b,&modulo));
```
Comment (gbenattar) [inline src/arithmetic/big_gmp.rs]: (minor): add a space after ",":
```
 assert_eq!(res, Mpz::mod_sub(&a, &b, &modulo));
```

Comment (gbenattar) [reply src/arithmetic/big_gmp.rs]: Same comment for all the changes.

**Comment by omershlo** [comment]
I fixed it

**Comment by gbenattar** [comment]
Sorry @omershlo I know I am being very picky here. Ideally after each "," you want a space assuming something follow. See this for more info: https://github.com/rust-lang-nursery/fmt-rfcs/blob/master/guide/items.md.

I am merging your code because it is just a styling issue.

**Pull request merged**